pkg/network/tracer/connection/perf_batching.go

```diff
@@ -177,16 +176,16 @@ func (p *perfBatchManager) cleanupExpiredState(now time.Time) {
 
 func newConnBatchManager(mgr *manager.Manager) (*perfBatchManager, error) {
 	connCloseEventMap, _, err := mgr.GetMap(probes.ConnCloseEventMap)
-	if err != nil {
-		return nil, err
+	if err != nil || connCloseEventMap == nil {
```

Comment:
If `err == nil` would be ever get a `nil` map? Also in the error below, if `err == nil` you will likely get a `<nil`> or something similar in the error message.

Reply:
[There's a code path in Manager.GetMap](https://github.com/DataDog/ebpf-manager/blob/2d6e96dfe1ddd1a62970b4e47bdd08e5d135b078/manager.go#L217) where the map could be nil with err == nil. We should probably change that in the manager I think

Reply:
Can we at least fix the error returned returned here in this case (for when `err == nil`?

Reply:
Yep, done!